deviceadvisor/script/DATestRun.py

```diff
@@ -0,0 +1,226 @@
+import boto3
+import uuid
+import json
+import os
+import subprocess
+import platform
+from time import sleep
+
+##############################################
+# Cleanup Certificates and Things and created certificate and private key file
+def delete_thing_with_certi(thingName, certiId, certiArn):
```

Comment:
As per the other PR(s) ,let's try and figure out a way to minimize duplicated, complex code across SDK repos.